README.md

```diff
@@ -173,10 +173,10 @@ custom `InitMsg` and `HandleMsg` structs for parsing your custom message types
 
 ```rust
 pub fn init<T: Storage>(store: &mut T, params: Params, msg: Vec<u8> ->
-  Result<Vec<CosmosMsg>, Error> { }
+  StdResult<Vec<CosmosMsg>> { }
```

Comment:
This is now `StdResult<InitResponse>` correct?

Reply:
Oof, this seems totally outdated...

Reply:
Jap, outdated beyond hope. But at least the current change is implemented :)